pkg/network/ebpf/manager.go

```diff
@@ -46,10 +46,10 @@ func NewManager(closedHandler *ebpf.PerfHandler, runtimeTracer bool) *manager.Ma
 			{Name: string(probes.UdpPortBindingsMap)},
 			{Name: "pending_bind"},
 			{Name: string(probes.TelemetryMap)},
+			{Name: string(probes.DoSendfileArgsMap)},
```

Comment:
Not a big deal, but curious why the order changed here?

Reply:
I did change the order to match (hand check to not miss something) with the output of the go script that generate the dump switch cases
But I will fallback to normal (remove noise)